src/routes/recipe.ts

```diff
@@ -38,4 +40,17 @@ router.get(
   validate,
   getRecipesLikeByUserId
 );
+router.delete("/recipe/:id", ({ params }, res) => {
```

Comment:
The base route for recipe is already prefixed with `/recipe` so there isn't a need to add it again (you can see this in `server.ts` line 61). All we have to do is add on to it.

Adding `/recipe/:id` here means the route is essentially this:
`localhost:5000/api/recipe/recipe/:id` 

Also we'll need to add the `protectedRoute` middleware (so only authenticated users can access this route)
```suggestion
router.delete("/:id", protectedRoute, ({ params }, res) => {
```

Reply:
You can also insert a validator after `protectedRoute` that validates the `req.params.id` There is only already defined in the validators file called `getParamIdValidator`. You can put it here and invoke it `()`. Don't forget to add `validate` afterwards so the validators work properly.